apps/web/pages/dashboard/[id]/teams.tsx

```diff
@@ -70,5 +70,6 @@ const Page: NextPageWithLayout = () => (
 );
 
 Page.Layout = DashLayout;
+Page.RequireAuth = true;
```

Comment:
now onwards we need to explicitly type the route protection of page this is a required field